test/system_test.js

```diff
@@ -1,70 +0,0 @@
-'use strict';
```

Comment:
the point on it for this is so that it makes sure whatever is currently in can be parsed, since that's the point of the class, and if something is missed, it is a consideration that should fail this.